.semaphore/semaphore.yml

```diff
@@ -32,7 +32,6 @@ global_job_config:
       - export BUILD_NUMBER=$(echo $SEMAPHORE_WORKFLOW_ID | cut -f1 -d"-")
       - export BRANCH_TAG=$(echo $SEMAPHORE_GIT_BRANCH | tr / -)
       # For PR Builds using Packaging
-      - pip install confluent-release-tools
```

Comment:
The desc says "replace release-tools with confluent-devtools" but there's only one removal of `confluent-release-tools` (without the replacement of `confluent-devtools`) in this PR. Is this intentional?

Reply:
yes, the confluent-devtools has already been preinstalled on CI build agent.